web/client/plugins/Identify.jsx

```diff
@@ -38,7 +40,7 @@ const Message = require('./locale/Message');
 require('./identify/identify.css');
 
 const selector = createStructuredSelector({
-    enabled: (state) => state.mapInfo && state.mapInfo.enabled || state.controls && state.controls.info && state.controls.info.enabled || false,
+    enabled: (state) => (state.mapInfo && state.mapInfo.enabled || state.controls && state.controls.info && state.controls.info.enabled) || state.mapInfo && state.mapInfo.enabled,
```

Comment:
This change does not seems to make sense.
(A OR B ) OR A = A OR B = A OR B OR false 

At least becomes undefined if (state.mapInfo === undefined && state.controls === undefined)
```suggestion
    enabled: (state) => (state.mapInfo && state.mapInfo.enabled || state.controls && state.controls.info && state.controls.info.enabled) || false,
```